.travis.yml

```diff
@@ -13,7 +13,7 @@ install: "pip install -r requirements.txt"
 before_script:
   - python2 testing/test-health-checks.py
   - ssh-keygen -N '' -f ~/.ssh/id_rsa && eval $(ssh-agent) && ssh-add ~/.ssh/id_rsa
-  - ln -sf $ANSIBLE_PLAYBOOK terraform.yml
+  - ln -sf $ANSIBLE_PLAYBOOK mantl.yml
```

Comment:
Yeah, I don't think we have much need to copy it before we start running multiple playbooks (which is probably far off). Maybe we can create an issue for that and either close or merge this?